### DISCUSSION TITLE
Full Screen Mode

### DISCUSSION THREAD

**Discussion by devDurgeshK** [Akascape/Easy-Tier-List#2] [Ideas]
## What is the Feature ?
Add a full screen mode that only shows the tier list and the contents to be arranged in a tier-list.

## How & Who it will help ?
It will help the content creators for recording the their tier list videos. As now the app is in the specific dimension, they would find another option for tier list as it would not look go on the video. But if the application is in the Full Screen Mode it will help them present the Tier list more easily.

---
and also a small request. the current font is the default font it would be great to change the font.

and i am really willing to work with you. If you like that pls contant me on discord(mcdk008) or on instagram(its_durgesh.7)

**Comment by Akascape** [comment]
@devDurgeshK Done, added fullscreen mode, font options and more. Check version 1.1
